pages/docs/relayer/running-relayer/quick-start.mdx

```diff
@@ -0,0 +1,192 @@
+
+  2023-03-14T14:13:08.316568Z DEBUG webb_relayer_config::cli: Using temp dir for store
+    at crates/relayer-config/src/cli.rs:131
+
+  2023-03-14T14:13:08.352455Z  INFO webb_relayer::service: Starting the server on 0.0.0.0:9955
+    at services/webb-relayer/src/service.rs:141
+
+  2023-03-14T14:13:08.355367Z DEBUG webb_relayer::service: Starting Background Services for (goerli) chain.
+    at services/webb-relayer/src/service.rs:172
+
+  2023-03-14T14:13:08.880637Z  WARN webb_relayer::service: Tx Queue disabled for (5)
+    at services/webb-relayer/src/service.rs:1053
+
+  2023-03-14T14:13:08.880641Z DEBUG webb_relayer::service: VAnchor events watcher for (0x9678…5077) Started.
+    at services/webb-relayer/src/service.rs:648
+
+  2023-03-14T14:13:08.880660Z DEBUG webb_probe: kind: lifecycle, started: true
+    at services/webb-relayer/src/main.rs:80
+
+  2023-03-14T14:13:08.880666Z  WARN webb_relayer::service: Governance relaying is not enabled for relayer
+    at services/webb-relayer/src/service.rs:1197
+
+  2023-03-14T14:13:16.286918Z  INFO webb_event_watcher_traits::evm::event_watcher: 🔄 #8650987 of #8654131 (99.9637%)
+    at crates/event-watcher-traits/src/evm/event_watcher.rs:218
+    in webb_event_watcher_traits::evm::event_watcher::run with chain_id: Ok(5), address: 0x9678…5077, tag: VAnchor Contract Watcher
+
+  2023-03-14T14:13:23.496016Z  INFO webb_event_watcher_traits::evm::event_watcher: 🔄 #8653887 of #8654131 (99.9972%)
+    at crates/event-watcher-traits/src/evm/event_watcher.rs:218
+    in webb_event_watcher_traits::evm::event_watcher::run with chain_id: Ok(5), address: 0x9678…5077, tag: VAnchor Contract Watcher
+
+  2023-03-14T14:13:39.302727Z  INFO webb_event_watcher_traits::evm::event_watcher: 🔄 #8654129 of #8654131 (100.0000%)
+    at crates/event-watcher-traits/src/evm/event_watcher.rs:218
+    in webb_event_watcher_traits::evm::event_watcher::run with chain_id: Ok(5), address: 0x9678…5077, tag: VAnchor Contract Watcher
+
+```
+
+### Verifying the Relayer is Running
+
+To verify that the relayer is running, you can use the `/api/v1/ip` endpoint:
+
+<Tabs items={["Running Locally", "on a server"]} storageKey="selected-binary">
+  <Tab>
+
+    ```bash
+    curl http://localhost:9955/api/v1/ip
+    ```
+
+  </Tab>
+  <Tab>
+
+    ```bash
+    curl http://<your-server-ip>:9955/api/v1/ip
+    ```
+
+  </Tab>
+</Tabs>
+
+You should see the following output:
+
+```json
+{"ip":"<your-public-ip>"}
+```
+
+**Congratulations**! You have successfully started the relayer.
+
+## What's Next?
+
+Now that you have the relayer running, you need to run it behind a reverse proxy. You can use [Caddy](https://caddyserver.com/) or [Nginx](https://www.nginx.com/) for this.
+
+Once you have the relayer running behind a reverse proxy, you should be able to access it from your domain name. For example, if you are running the relayer on `relayer.example.com`, you should be able to access the relayer at `https://relayer.example.com/api/v1/ip`.
```

Comment:
We should provide a guide for doing this.

Reply:
Do you want me to make it in the next step? or the same page? I would prefer a separate tutorial/page, since I would like to keep the quick start short and quick ofc.